exporter/newrelicexporter/README.md

```diff
@@ -40,13 +40,13 @@ exporters:
     timeout: 30s
 
   # host_override is set to send data to New Relic's EU data centers.
```

Comment:
```suggestion
    # host_override is set to send data to New Relic's EU data centers.
```
Might as well indent the comment as well 😄

Reply:
The CLA is signed and the above change is made. Thanks for your review